src/routes/healthcheck.ts

```
@@ -0,0 +1,88 @@
+import { Router, type Request, type Response } from "express";
+import logger from "../utils/logger";
```

ESLint

[error] Insert `{·createNotificationClient·}·from·"../notifications/client";⏎import·`

(prettier/prettier)